src/components/AppSwitch.js

```diff
@@ -73,7 +73,13 @@ class AppSwitch extends React.Component<LoadingProps, {}> {
 
       if (jwt) {
         log.debug('New account, not verified, or did not finish signup', jwt)
-        this.props.navigation.navigate('Auth')
+
```

Comment:
I prefer to use `const { navigation, store } = this.props` at the beginning of the method, so it's less text, but I think it's just a nice to have, haha